src/components/views/messages/MPollBody.tsx

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
```

Comment:
I give up on the rabbit hole it keeps spitting errors for in adajacent code, https://github.com/matrix-org/matrix-react-sdk/actions/runs/3424467797/jobs/5704176614

I think I've wrapped up everything relevant to the `getRelationsForEvent` changes ⏩

Reply:
The strict TypeScript job is ok to fail as it's not "Required" but this detail is easy to miss. Merging with CI red isn't something I'm going to do willy nilly to understand that detail. I'm also diligent enough to try to get CI to pass by default.

> Aside: It's totally unclear that tsc strict linter is allowed to fail (until next week) which spawned these changes. I suppose this is mostly because GitHub actions don't have an "allowed-to-fail" behavior where the icon changes to `!`. But I assume we could switch it to pass and still add the lints to the diff. And would be nice to echo out a message in that job that these are allowed to fail or "not absolutely necessary to make this happy" (add some ANSI color to make it stand out at the top and bottom). But this point doesn't even matter because the strict TypeScript checks will become required next week anyway so this change would happen if I just waited another week.
> 
> The `strict` errors are also extremely hard to diagnose locally since there it's not in any config or `package.json` `script` to run. The solution here is better tooling or solving all of the `strict` errors on `develop` so we don't run into unrelated problems to the PR at hand. @robintown mentioned installing `typescript-strict-plugin` but it doesn't have the diff comparison and unclear how easy to turn it on/off.
>
> *-- @MadLittleMods, https://github.com/matrix-org/matrix-react-sdk/pull/9558#discussion_r1018307810*

The important part in that text is that "this point doesn't even matter because the strict TypeScript checks will become required next week anyway so this change would happen if I just waited another week."

Reply:
Discussed in the [Web Weekly meeting today](https://docs.google.com/document/d/1wa1jGYsN-rOzvQB4tky5e3lwy3jl5PNEVneALwwg8XM/edit#bookmark=id.eqcq6590kpfi) and fixed up the [last lint](https://github.com/matrix-org/matrix-react-sdk/pull/9558#discussion_r1019643241) that was in a changed part of this PR (still adjacent but the line was changed)

Reply:
Strict type check has gone especially wild in this last run 🤷, https://github.com/matrix-org/matrix-react-sdk/actions/runs/3440677821/jobs/5739409413

But shouldn't matter ⏩